catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -726,40 +726,51 @@ const useDropzoneMessageStyles = M.makeStyles((t) => ({
 
 interface DropzoneMessageProps {
   error: React.ReactNode
-  warn: { upload: boolean; s3: boolean }
+  warn: { upload: boolean; s3: boolean; count: boolean }
 }
 
 function DropzoneMessage({ error, warn }: DropzoneMessageProps) {
   const classes = useDropzoneMessageStyles()
 
   const label = React.useMemo(() => {
     if (error) return error
-    if (warn.upload || warn.s3)
-      return (
+    const warnings: React.ReactNode[] = []
+    const push = (key: string, warning: React.ReactNode) => {
```

Comment:
It looks sophisticated at first sight. Maybe, make it simpler like this:

```tsx
// replace span with div
// use <p></p> instead of <fragment></fragment><br />

<div class={root}>
  <>
    {warn.upload && <p key="upload">Error {MAX_UPLOAD}</p>}
    {warn.s3 && <p key="upload">Error {MAX_S3_SIZE}</p>}
    {warn.count && <p key="upload">Error {FILE_COUNT}</p>}
  </>
</div>

Reply:
ok makes sense